euler/README.md

```diff
@@ -0,0 +1,263 @@
+
+- create a set of "samples" in the samples folder, i. e. a set of parameter configurations that are to be simulated
+- run a demand simulation for each sample and store the output to the samples folder
+  - copy the reference case folder to the simulation folder
+  - apply sample parameters to the reference case
+- merge the output data back to a single dataset
+- run analysis on the output data, generating the output file "analysis.xls" in the samples folder
+
+## File locations
+
+The bash scripts make some assumptions on the installation of the CEA:
+
+- the CEAforArcGIS repository is cloned to `$HOME/CEAforArcGIS`
+- the default reference case is found in `$HOME/cea-reference-case/reference-case-open/baseline`
+  - note: since git-lfs is not installed on Euler, you will need to manually copy the radiation files to the cluster 
+    (`radiation.csv`, `properties_surfaces.csv`)
+
+
+The Euler cluster has [different filesystems](https://scicomp.ethz.ch/wiki/Data_management) for different purposes. 
+Due to the way the simulation tasks are split up, the simulation uses the following folders:
+
+- the reference case (`--scenario-path`) is used as a copy of the original reference case - each 
+  sample is created from this case as a copy. You can keep the reference case anywhere you like in your 
+  home folder or personal scratch storage.
+  - home folder: `/cluster/home/username` (backed up, permanent, alias: `$HOME`)
+  - personal scratch storage: `/cluster/scratch/username` (not backed up, deleted after 15 days, alias: `$SCRATCH`)
+  
+- the weather path: the path to the *.epw file used for simulation. This can be kept in the home folder.
+    - the default weather file used by the bash scripts is `$HOME/CEAforArcGIS/cea/databases/CH/Weather/Zurich.epw`
+  
+- the samples folder contains the inputs (list of samples and the problem statement) and the final outputs of the
+  analysis. This should be stored in your personal scratch folder (as the home folder has a limit to 100k files).
+    - the samples folder used by the bash scripts is `${SCRATCH}/samples_${METHOD}_${N}` (e.g. 
+      `/cluster/scratch/darthoma/samples_morris_100`)    
+  
+- the simulation reference case is a copy of the original reference case with the variables from the sampling method
+  applied to the input data. This folder is created by the demand simulation script and is stored in the local
+  scratch folder of the node running the demand simulation.
+  - local scratch storage: `/scratch` 
+  - (actually, the bash scripts use the `$TMPDIR` variable to ensure simultaneous simulations on the same node don't step on 
+    each other's feet)
+    
+## Running the scripts
+
+There are two ways to run the scripts. The first way is with the python modules as shown for the Windows case. On
+Euler, the scripts in the `euler` folder of the `CEAforArcGIS` repository contains a set of handy bash scripts that
```

Comment:
@daren-thomas this scripts are fantastic. however, do i have to re clone the repository every time i make a change in github? or do i use nano to change the scripts variables?

in the run demand.sh what is NUM_SIMULATIONS=${NUM_SIMULATIONS:-100}?